homework/content.js

```diff
@@ -1,37 +1,44 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+var ContentManager =(function() {
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+  // constructor
+  var self = function () {
+    this._wrapper = null; 
+  };
+ 
+  // prototype
+  self.prototype = {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+    start() {
+      this._wrapper = document.querySelector('#note-content-wrapper');
```

Comment:
Instead of using an anonymous function, setting the listener as this and handle events with handleEvent method is better.